dace/codegen/codegen.py

```diff
@@ -98,6 +99,8 @@ def generate_code(sdfg) -> List[CodeObject]:
     # Before compiling, validate SDFG correctness
     sdfg.validate()
 
+    MapPython(sdfg.to_json(), sdfg.name)
```

Comment:
1. `sdfg.to_json()` is an expensive operation, easier if `sdfg` is used directly.
2. operation happening in class constructor is a weird behavior
3. it ends up staying unused, as the object is not assigned to anything?